.github/workflows/ci.yml

```diff
@@ -25,21 +25,21 @@ jobs:
     - name: Set up Go
       uses: actions/setup-go@v2
       with:
-        go-version: 1.16.*
+        go-version: 1.18.*
```

Comment:
I recall we had some discussion about this when we did the last release, but could you change the go-versions to have the exact version? (`1.18.8`); that way it's (also when doing releases) transparent "what version was used" for a release, instead of "what happened to be the version at the time".

Reply:
updated.